### PR TITLE
WIP: Issue #700 - Not working in Bitbucket pipeline: "authorization denied…

### PR DESCRIPTION
**Pull request by 4ntoine**
… by plugin pipelines: Command not supported."

Detecting Bitbucket build and creating temporary files/dirs within BITBUCKET_CLONE_DIR.

### CODE REVIEW

File: core/src/main/java/org/testcontainers/utility/MountableFile.java [hunk 1]
```diff
@@ -241,8 +243,23 @@ private String extractClassPathResourceToTempLocation(final String hostPath) {
         return tmpLocation.getAbsolutePath();
     }
 
+    private boolean isOnBitbucket() {
+        final String bbCloneDir = System.getenv(BITBUCKET_CLONE_DIR_ENV);
```
Comment (dbyron0) [inline core/src/main/java/org/testcontainers/utility/MountableFile.java]: call getBitbucketCloneDir() here?

Comment (4ntoine) [reply core/src/main/java/org/testcontainers/utility/MountableFile.java]: probably some other vars will be required so i'd prefer not to join BB checking and getting some BB variable

**Comment by 4ntoine** [comment]
Tested on bitbucket (building my fork of testcontainers in the very same pipeline to install it to maven local and then require it while building my module). Still not workingdue to the same error output  ... To be continued.

**Comment by rnacher** [comment]
Just to add more light to this issue, I think that the problem still continues because it is related to the ryuk container used to "handle" all other containers. Take a look at  ResourceReaper.class:

`binds.add(new Bind("//var/run/docker.sock", new Volume("/var/run/docker.sock")));`

and this is forbidden by BBPipelines. I tryed to update it to use the` tcp://hostIpAddress:2375` instead but, it didn't work either.

**Comment by erikvanzijst** [comment]
Any update on this?

**Comment by bsideup** [comment]
AFAIK BitBucket implemented a solution on their side and this PR is obsolete. 
But feel free to re-open it if I'm wrong :)

**Pull request closed without merging**